In [2]:
import numpy as np
import matplotlib.pyplot as plt
from scipy import stats
import pandas as pd
import warnings 
import statistics as sts
warnings.filterwarnings('ignore')

In [3]:
df = pd.read_csv('./Amazon_bestsellers_items_2025.csv')
df

,Unnamed: 0,rank,asin,product_title,product_price,product_star_rating,product_num_ratings,product_url,product_photo,rank_change_label,country,page
0,0,1,B073VKKNN9,Kaspersky | Premium - Total Security (Ultimate...,₹469.00,4.3,13324.0,https://www.amazon.in/dp/B073VKKNN9,https://images-eu.ssl-images-amazon.com/images...,NaN,IN,1
1,1,2,B07PQZJ6Y8,"K7 Security K7, Total Security, 1 User, 1 Year...",₹370.00,4.4,2291.0,https://www.amazon.in/dp/B07PQZJ6Y8,https://images-eu.ssl-images-amazon.com/images...,NaN,IN,1
2,2,3,B0D1KL34JM,Microsoft Office 2021 Professional - Lifetime ...,"₹1,799.00",4.5,388.0,https://www.amazon.in/dp/B0D1KL34JM,https://images-eu.ssl-images-amazon.com/images...,NaN,IN,1
3,3,4,B07B9YYLGG,"Bitdefender - 1 Device,1 Year - Mobile Securit...",₹94.00,4.1,9630.0,https://www.amazon.in/dp/B07B9YYLGG,https://images-eu.ssl-images-amazon.com/images...,NaN,IN,1
4,4,5,B073VLGMZ4,"McAfee Total Protection 2025 | 1 Device, 3 Yea...","₹1,699.00",4.4,5783.0,https://www.amazon.in/dp/B073VLGMZ4,https://images-eu.ssl-images-amazon.com/images...,NaN,IN,1
...,...,...,...,...,...,...,...,...,...,...,...,...
994,994,96,B0DP7LBGM1,Microsoft Windows Server 2025 Standard Edition...,"$1,561.01",NaN,NaN,https://www.amazon.com.mx/dp/B0DP7LBGM1,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
995,995,97,B0CFZNMZW6,Photoshop Elements 2024 and Premiere Elements ...,"$3,067.45",4.0,82.0,https://www.amazon.com.mx/dp/B0CFZNMZW6,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
996,996,98,B09H2BRVWZ,Microsoft Office Hogar y Empresa 2021,"$5,999.00",4.9,12.0,https://www.amazon.com.mx/dp/B09H2BRVWZ,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
997,997,99,B0C55NKHKL,"Virtual DJ and Karaoke Studio 8, Lifetime, 1 D...",NaN,3.7,3.0,https://www.amazon.com.mx/dp/B0C55NKHKL,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2


In [4]:
df_Mexico = df[df['country'] == "MX"]
df_Mexico

,Unnamed: 0,rank,asin,product_title,product_price,product_star_rating,product_num_ratings,product_url,product_photo,rank_change_label,country,page
899,899,1,B09FR4MTK8,Microsoft 365 Familia (12 Meses) - Standard - ...,"$1,929.00",4.6,469.0,https://www.amazon.com.mx/dp/B09FR4MTK8,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
900,900,2,B09FRYNWPD,Microsoft 365 Personal (12 Meses) - Standard -...,"$1,569.00",4.6,471.0,https://www.amazon.com.mx/dp/B09FRYNWPD,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
901,901,3,B0DJPWBBJV,Microsoft Office Home 2024 | Apps clásicas: Wo...,"$2,649.00",4.4,22.0,https://www.amazon.com.mx/dp/B0DJPWBBJV,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
902,902,4,B086XTW6WF,Microsoft 365 Personal | Suscripción anual | P...,"$1,770.00",4.7,947.0,https://www.amazon.com.mx/dp/B086XTW6WF,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
903,903,5,B0BS9MY1TV,Microsoft 365 Familia Extra Time 2022 (12 Mese...,NaN,4.6,175.0,https://www.amazon.com.mx/dp/B0BS9MY1TV,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
...,...,...,...,...,...,...,...,...,...,...,...,...
994,994,96,B0DP7LBGM1,Microsoft Windows Server 2025 Standard Edition...,"$1,561.01",NaN,NaN,https://www.amazon.com.mx/dp/B0DP7LBGM1,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
995,995,97,B0CFZNMZW6,Photoshop Elements 2024 and Premiere Elements ...,"$3,067.45",4.0,82.0,https://www.amazon.com.mx/dp/B0CFZNMZW6,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
996,996,98,B09H2BRVWZ,Microsoft Office Hogar y Empresa 2021,"$5,999.00",4.9,12.0,https://www.amazon.com.mx/dp/B09H2BRVWZ,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
997,997,99,B0C55NKHKL,"Virtual DJ and Karaoke Studio 8, Lifetime, 1 D...",NaN,3.7,3.0,https://www.amazon.com.mx/dp/B0C55NKHKL,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2


In [19]:
df_MX_Microsoft = df_Mexico[df_Mexico['product_title'].str.contains("Microsoft")]
df_MX_Microsoft


,Unnamed: 0,rank,asin,product_title,product_price,product_star_rating,product_num_ratings,product_url,product_photo,rank_change_label,country,page
899,899,1,B09FR4MTK8,Microsoft 365 Familia (12 Meses) - Standard - ...,"$1,929.00",4.6,469.0,https://www.amazon.com.mx/dp/B09FR4MTK8,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
900,900,2,B09FRYNWPD,Microsoft 365 Personal (12 Meses) - Standard -...,"$1,569.00",4.6,471.0,https://www.amazon.com.mx/dp/B09FRYNWPD,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
901,901,3,B0DJPWBBJV,Microsoft Office Home 2024 | Apps clásicas: Wo...,"$2,649.00",4.4,22.0,https://www.amazon.com.mx/dp/B0DJPWBBJV,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
902,902,4,B086XTW6WF,Microsoft 365 Personal | Suscripción anual | P...,"$1,770.00",4.7,947.0,https://www.amazon.com.mx/dp/B086XTW6WF,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
903,903,5,B0BS9MY1TV,Microsoft 365 Familia Extra Time 2022 (12 Mese...,NaN,4.6,175.0,https://www.amazon.com.mx/dp/B0BS9MY1TV,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
911,911,13,B07DYKZ9MG,CLIP STUDIO PAINT EX - Version 1 - for Microso...,"$1,492.10",4.4,275.0,https://www.amazon.com.mx/dp/B07DYKZ9MG,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
913,913,15,B086XTJSPF,Microsoft 365 Familia | Suscripción anual | Pa...,"$2,299.00",4.8,1141.0,https://www.amazon.com.mx/dp/B086XTJSPF,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
914,914,16,B09MYBD79G,Microsoft OEM System Builder | Windоws 11 Pro ...,"$3,490.00",4.1,783.0,https://www.amazon.com.mx/dp/B09MYBD79G,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
917,917,19,B0BS9VJHWQ,Microsoft 365 Personal Extra Time 2022 (12 Mes...,NaN,4.4,42.0,https://www.amazon.com.mx/dp/B0BS9VJHWQ,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
919,919,21,B06XNJB66F,LibreOffice 2025 compatible con Microsoft Offi...,$399.99,4.1,653.0,https://www.amazon.com.mx/dp/B06XNJB66F,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1


Podemos observar que hay 27 productos de la familia Microsoft en el top 100 de los productos mejor vendidos en México

In [6]:
df_MX_Microsoft_Rating = df_MX_Microsoft[df_MX_Microsoft['product_star_rating']>=4.7]
df_MX_Microsoft_Rating

,Unnamed: 0,rank,asin,product_title,product_price,product_star_rating,product_num_ratings,product_url,product_photo,rank_change_label,country,page
902,902,4,B086XTW6WF,Microsoft 365 Personal | Suscripción anual | P...,"$1,770.00",4.7,947.0,https://www.amazon.com.mx/dp/B086XTW6WF,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
913,913,15,B086XTJSPF,Microsoft 365 Familia | Suscripción anual | Pa...,"$2,299.00",4.8,1141.0,https://www.amazon.com.mx/dp/B086XTJSPF,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
924,924,26,B09DGZMX7K,Microsoft Office - Hogar y Estudiante - 1 Disp...,NaN,4.7,113.0,https://www.amazon.com.mx/dp/B09DGZMX7K,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
932,932,34,B09H2N99B4,Microsoft 365 Familia 2021 | Suscripción anual...,"$2,299.00",4.9,147.0,https://www.amazon.com.mx/dp/B09H2N99B4,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
933,933,35,B004ZLUZIM,Microsoft 365 Personal 2021 | Suscripción anua...,"$1,829.00",4.8,90.0,https://www.amazon.com.mx/dp/B004ZLUZIM,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
975,975,77,B09DH2LCX5,Microsoft Office - Hogar y Empresa - 1 Disposi...,NaN,4.7,113.0,https://www.amazon.com.mx/dp/B09DH2LCX5,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
996,996,98,B09H2BRVWZ,Microsoft Office Hogar y Empresa 2021,"$5,999.00",4.9,12.0,https://www.amazon.com.mx/dp/B09H2BRVWZ,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2


Al hacer un filtrado de los datos con ayuda del código anterior podemos notar que de los 27 productos obtenidos de la familia Microsoft, solo hay 7 que tienen una calificación mayor a 4.7

In [7]:
df_MX_Microsoft_Rating['product_num_ratings'].describe().to_frame()


,product_num_ratings
count,7.000000
mean,366.142857
std,468.269306
min,12.000000
25%,101.500000
50%,113.000000
75%,547.000000
max,1141.000000


En la tabla superior podemos encontrar que los productos de la familia Microsoft que se encuentran en el top de productos más vendidos en México, reciben una media de comentarios o de calificaciones respecto a su calidad de 366, pero hay que tener en cuenta que esta media esta siendo afectada por valores extremos debido a que el mínimo de reseñas recibidas es de 12 mientras que el máximo es de 1141, por lo que podemos notar una gran diferencia en esto

In [8]:
df_MX_Microsoft_Rating['product_price'] = (df_MX_Microsoft_Rating['product_price'].str.replace(r"[^0-9.]", "", regex=True).astype(float))
df_MX_Microsoft_Rating

,Unnamed: 0,rank,asin,product_title,product_price,product_star_rating,product_num_ratings,product_url,product_photo,rank_change_label,country,page
902,902,4,B086XTW6WF,Microsoft 365 Personal | Suscripción anual | P...,1770.0,4.7,947.0,https://www.amazon.com.mx/dp/B086XTW6WF,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
913,913,15,B086XTJSPF,Microsoft 365 Familia | Suscripción anual | Pa...,2299.0,4.8,1141.0,https://www.amazon.com.mx/dp/B086XTJSPF,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
924,924,26,B09DGZMX7K,Microsoft Office - Hogar y Estudiante - 1 Disp...,NaN,4.7,113.0,https://www.amazon.com.mx/dp/B09DGZMX7K,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
932,932,34,B09H2N99B4,Microsoft 365 Familia 2021 | Suscripción anual...,2299.0,4.9,147.0,https://www.amazon.com.mx/dp/B09H2N99B4,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
933,933,35,B004ZLUZIM,Microsoft 365 Personal 2021 | Suscripción anua...,1829.0,4.8,90.0,https://www.amazon.com.mx/dp/B004ZLUZIM,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
975,975,77,B09DH2LCX5,Microsoft Office - Hogar y Empresa - 1 Disposi...,NaN,4.7,113.0,https://www.amazon.com.mx/dp/B09DH2LCX5,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
996,996,98,B09H2BRVWZ,Microsoft Office Hogar y Empresa 2021,5999.0,4.9,12.0,https://www.amazon.com.mx/dp/B09H2BRVWZ,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2


In [9]:
df_MX_Microsoft_Rating['product_price'].describe().to_frame()


,product_price
count,5.000000
mean,2839.200000
std,1784.072644
min,1770.000000
25%,1829.000000
50%,2299.000000
75%,2299.000000
max,5999.000000


Obtenemos la información del precio de los productos mejor calificados de Microsoft, y encontramos que a pesar de que el promedio de precios de estos productos es de $2,839.2$ pesos, sin embargo, al igual que el caso de la cantidad de reseñas de estos productos, este resultado esta siendo afectado por valores extremos, esto debido a que el precio mínimo es de $1,770$ pesos y el precio máximo es de $5,999.0$ pesos, y es notable la gran diferencia entre estos valores.

In [10]:
df_MX_top = df_Mexico[df_Mexico['rank']<=10]
df_MX_top

,Unnamed: 0,rank,asin,product_title,product_price,product_star_rating,product_num_ratings,product_url,product_photo,rank_change_label,country,page
899,899,1,B09FR4MTK8,Microsoft 365 Familia (12 Meses) - Standard - ...,"$1,929.00",4.6,469.0,https://www.amazon.com.mx/dp/B09FR4MTK8,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
900,900,2,B09FRYNWPD,Microsoft 365 Personal (12 Meses) - Standard -...,"$1,569.00",4.6,471.0,https://www.amazon.com.mx/dp/B09FRYNWPD,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
901,901,3,B0DJPWBBJV,Microsoft Office Home 2024 | Apps clásicas: Wo...,"$2,649.00",4.4,22.0,https://www.amazon.com.mx/dp/B0DJPWBBJV,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
902,902,4,B086XTW6WF,Microsoft 365 Personal | Suscripción anual | P...,"$1,770.00",4.7,947.0,https://www.amazon.com.mx/dp/B086XTW6WF,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
903,903,5,B0BS9MY1TV,Microsoft 365 Familia Extra Time 2022 (12 Mese...,NaN,4.6,175.0,https://www.amazon.com.mx/dp/B0BS9MY1TV,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
904,904,6,B0B5T2J8TG,Corel PaintShop Pro 2023 Ultimate | Potente so...,"$2,221.85",4.0,157.0,https://www.amazon.com.mx/dp/B0B5T2J8TG,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
905,905,7,B0D2Y8LLYV,DVD For Windows Emergency Boot Disk CD for Win...,$237.02,NaN,NaN,https://www.amazon.com.mx/dp/B0D2Y8LLYV,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
906,906,8,B0DYGB53TC,Punto de Venta Papelería,$690.00,NaN,NaN,https://www.amazon.com.mx/dp/B0DYGB53TC,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
907,907,9,B0DVQWCBZL,Programa Punto De Venta Sistema,$690.00,NaN,NaN,https://www.amazon.com.mx/dp/B0DVQWCBZL,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
908,908,10,B09VPVXWWC,Mezclador Digital Estéreo de 4 Canales Y Mezcl...,$658.00,5.0,5.0,https://www.amazon.com.mx/dp/B09VPVXWWC,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1


In [20]:
df_MX_top['product_num_ratings'].describe().to_frame()

,product_num_ratings
count,7.000000
mean,320.857143
std,335.099204
min,5.000000
25%,89.500000
50%,175.000000
75%,470.000000
max,947.000000


Podemos observar que de los 10 producgtos más vendidos de México, tienen un promedio de numero de reseñas en estos productos es de 320, sin embargo, hay que tener en cuenta que los valores extremos están afectando a este calculo, ya que hay un mínimo de 5 reseñas y un máximo de 947

In [12]:
df_MX_top['product_price'] = (df_MX_top['product_price'].str.replace(r"[^0-9.]", "", regex=True).astype(float))
df_MX_top

,Unnamed: 0,rank,asin,product_title,product_price,product_star_rating,product_num_ratings,product_url,product_photo,rank_change_label,country,page
899,899,1,B09FR4MTK8,Microsoft 365 Familia (12 Meses) - Standard - ...,1929.00,4.6,469.0,https://www.amazon.com.mx/dp/B09FR4MTK8,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
900,900,2,B09FRYNWPD,Microsoft 365 Personal (12 Meses) - Standard -...,1569.00,4.6,471.0,https://www.amazon.com.mx/dp/B09FRYNWPD,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
901,901,3,B0DJPWBBJV,Microsoft Office Home 2024 | Apps clásicas: Wo...,2649.00,4.4,22.0,https://www.amazon.com.mx/dp/B0DJPWBBJV,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
902,902,4,B086XTW6WF,Microsoft 365 Personal | Suscripción anual | P...,1770.00,4.7,947.0,https://www.amazon.com.mx/dp/B086XTW6WF,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
903,903,5,B0BS9MY1TV,Microsoft 365 Familia Extra Time 2022 (12 Mese...,NaN,4.6,175.0,https://www.amazon.com.mx/dp/B0BS9MY1TV,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
904,904,6,B0B5T2J8TG,Corel PaintShop Pro 2023 Ultimate | Potente so...,2221.85,4.0,157.0,https://www.amazon.com.mx/dp/B0B5T2J8TG,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
905,905,7,B0D2Y8LLYV,DVD For Windows Emergency Boot Disk CD for Win...,237.02,NaN,NaN,https://www.amazon.com.mx/dp/B0D2Y8LLYV,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
906,906,8,B0DYGB53TC,Punto de Venta Papelería,690.00,NaN,NaN,https://www.amazon.com.mx/dp/B0DYGB53TC,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
907,907,9,B0DVQWCBZL,Programa Punto De Venta Sistema,690.00,NaN,NaN,https://www.amazon.com.mx/dp/B0DVQWCBZL,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1
908,908,10,B09VPVXWWC,Mezclador Digital Estéreo de 4 Canales Y Mezcl...,658.00,5.0,5.0,https://www.amazon.com.mx/dp/B09VPVXWWC,https://images-na.ssl-images-amazon.com/images...,NaN,MX,1


In [22]:
df_MX_top['product_price'].describe().to_frame()

,product_price
count,9.000000
mean,1379.318889
std,835.776400
min,237.020000
25%,690.000000
50%,1569.000000
75%,1929.000000
max,2649.000000


El promedio de precio de los 10 productos más vendidos en México es de 1379.31 pesos, sin embargo al igual que todos los casos anteriores, los valores extremos están afectando a esto ya que el valor minimo es de 237.02 pesos mientras que el valor máximo es de 2,649 pesos

In [14]:
df_MX_top_less = df_Mexico[df_Mexico['rank']>=90]
df_MX_top_less

,Unnamed: 0,rank,asin,product_title,product_price,product_star_rating,product_num_ratings,product_url,product_photo,rank_change_label,country,page
988,988,90,B010MIN9A2,ClearClick VHS to DVD Wizard Software with USB...,$719.95,4.0,1373.0,https://www.amazon.com.mx/dp/B010MIN9A2,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
989,989,91,B00NW2ZJB8,Nuance Dragon USB Headset,NaN,4.0,718.0,https://www.amazon.com.mx/dp/B00NW2ZJB8,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
990,990,92,B0CJYL7JYV,"TurboTax Deluxe 2023 Tax Software, Federal & S...",NaN,4.6,6407.0,https://www.amazon.com.mx/dp/B0CJYL7JYV,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
991,991,93,B0CM553HKF,Moho Debut 14 | Animation software for PC and ...,"$1,284.49",4.1,56.0,https://www.amazon.com.mx/dp/B0CM553HKF,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
992,992,94,B0DYGBP3RY,Software Punto de Venta Sistema - 2 Licencias,$999.00,NaN,NaN,https://www.amazon.com.mx/dp/B0DYGBP3RY,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
993,993,95,B0D2CZPL7T,CLIP STUDIO PAINT PRO - Version 3 | Perpetual ...,"$1,429.03",4.9,15.0,https://www.amazon.com.mx/dp/B0D2CZPL7T,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
994,994,96,B0DP7LBGM1,Microsoft Windows Server 2025 Standard Edition...,"$1,561.01",NaN,NaN,https://www.amazon.com.mx/dp/B0DP7LBGM1,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
995,995,97,B0CFZNMZW6,Photoshop Elements 2024 and Premiere Elements ...,"$3,067.45",4.0,82.0,https://www.amazon.com.mx/dp/B0CFZNMZW6,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
996,996,98,B09H2BRVWZ,Microsoft Office Hogar y Empresa 2021,"$5,999.00",4.9,12.0,https://www.amazon.com.mx/dp/B09H2BRVWZ,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
997,997,99,B0C55NKHKL,"Virtual DJ and Karaoke Studio 8, Lifetime, 1 D...",NaN,3.7,3.0,https://www.amazon.com.mx/dp/B0C55NKHKL,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2


In [25]:
df_MX_top_less['product_num_ratings'].describe().to_frame()

,product_num_ratings
count,9.000000
mean,1045.444444
std,2066.881776
min,3.000000
25%,15.000000
50%,82.000000
75%,743.000000
max,6407.000000


El numero de reseñas de los "peores" productos vendidos del top 100 en México tienen un promedio de 1045 reseñas, sin embargo esta siendo afectado por los valores extremos, la cantidad minima de reseñas que recibió un producto es de 3, mientras que el máximo es de 6,407 reseñas

In [16]:
df_MX_top_less['product_price'] = (df_MX_top_less['product_price'].str.replace(r"[^0-9.]", "", regex=True).astype(float))
df_MX_top_less

,Unnamed: 0,rank,asin,product_title,product_price,product_star_rating,product_num_ratings,product_url,product_photo,rank_change_label,country,page
988,988,90,B010MIN9A2,ClearClick VHS to DVD Wizard Software with USB...,719.95,4.0,1373.0,https://www.amazon.com.mx/dp/B010MIN9A2,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
989,989,91,B00NW2ZJB8,Nuance Dragon USB Headset,NaN,4.0,718.0,https://www.amazon.com.mx/dp/B00NW2ZJB8,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
990,990,92,B0CJYL7JYV,"TurboTax Deluxe 2023 Tax Software, Federal & S...",NaN,4.6,6407.0,https://www.amazon.com.mx/dp/B0CJYL7JYV,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
991,991,93,B0CM553HKF,Moho Debut 14 | Animation software for PC and ...,1284.49,4.1,56.0,https://www.amazon.com.mx/dp/B0CM553HKF,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
992,992,94,B0DYGBP3RY,Software Punto de Venta Sistema - 2 Licencias,999.00,NaN,NaN,https://www.amazon.com.mx/dp/B0DYGBP3RY,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
993,993,95,B0D2CZPL7T,CLIP STUDIO PAINT PRO - Version 3 | Perpetual ...,1429.03,4.9,15.0,https://www.amazon.com.mx/dp/B0D2CZPL7T,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
994,994,96,B0DP7LBGM1,Microsoft Windows Server 2025 Standard Edition...,1561.01,NaN,NaN,https://www.amazon.com.mx/dp/B0DP7LBGM1,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
995,995,97,B0CFZNMZW6,Photoshop Elements 2024 and Premiere Elements ...,3067.45,4.0,82.0,https://www.amazon.com.mx/dp/B0CFZNMZW6,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
996,996,98,B09H2BRVWZ,Microsoft Office Hogar y Empresa 2021,5999.00,4.9,12.0,https://www.amazon.com.mx/dp/B09H2BRVWZ,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2
997,997,99,B0C55NKHKL,"Virtual DJ and Karaoke Studio 8, Lifetime, 1 D...",NaN,3.7,3.0,https://www.amazon.com.mx/dp/B0C55NKHKL,https://images-na.ssl-images-amazon.com/images...,NaN,MX,2


In [26]:
df_MX_top_less['product_price'].describe().to_frame()

,product_price
count,8.000000
mean,1994.017500
std,1774.001344
min,719.950000
25%,972.302500
50%,1356.760000
75%,1937.620000
max,5999.000000


El precio promedio de los productos que se encuentran en los últimos lugares del top de productos más vendidos en México es de 1,994.01 pesos, pero al igual que los casos anteriores, esto esta siendo afectado por los valores extremos, el producto con el menor precio dentro de esta clasificación es de 719.95 pesos y el producto con el mayor precio es de 5,999 pesos